### ISSUE TITLE
[Bug] GridBot Fails On Order Object Return When Using BitGet

### ISSUE THREAD

**Issue by dragmecom** [hackingthemarkets/gridbot-websockets#5]
As can be seen here : 
https://pastebin.com/kb4T4YaR

When running with BitGet the script fails on line 50 and 75 because it fails to return the order object.

Also brought it up with BitGet's team themselves since it looks like a proper call and your code looks right but if you see something that is off, let me know, if you have an update!

**Comment by ketzah** [comment]
Yes it's broken somewhere in these buy/sell for loops (python version)

for buy_order in buy_orders:
      print("checking buy order {}".format(buy_order['id']))
      try:
           order = exchange.fetch_order(buy_order['id'])

It places orders correctly but order management is broken.
